Source/Core/DolphinQt2/GameList/GameTracker.h

```diff
@@ -34,14 +34,33 @@ class GameTracker final : public QFileSystemWatcher
   void GameRemoved(const QString& path);
 
 private:
-  void LoadGame(const QString& path);
   void UpdateDirectory(const QString& dir);
   void UpdateFile(const QString& path);
+  void AddDirectoryInternal(const QString& dir);
+  void RemoveDirectoryInternal(const QString& dir);
+  void UpdateDirectoryInternal(const QString& dir);
+  void UpdateFileInternal(const QString& path);
   QSet<QString> FindMissingFiles(const QString& dir);
+  void LoadGame(const QString& path);
+
+  enum class CommandType
+  {
+    LoadCache,
+    AddDirectory,
+    RemoveDirectory,
+    UpdateDirectory,
+    UpdateFile
```

Comment:
Any thoughts on adding a trailing comma here? That way, if we ever decide to add another command, the diff will be shorter and easier to read.